h/browser/chrome/karma.config.js

```diff
@@ -32,6 +41,9 @@ module.exports = function(config) {
       '../../static/scripts/blocklist.js'
     ],
 
+    proxies: {
+      "/settings.json": "http://localhost:9877/base/test/settings.json"
```

Comment:
The code expects to find `/settings.json` (which is where the file is in production), but Karma serves it at the wrong location, so proxy it.